src/reactapp/src/components/paypalExpress/hooks/usePaypalExpressCartContext.js

```diff
@@ -0,0 +1,36 @@
+import { useContext, useMemo } from 'react';
+import _get from 'lodash.get';
+import CartContext from '../../../context/Cart/CartContext';
+import AppContext from '../../../context/App/AppContext';
+
+export default function usePaypalExpressCartContext() {
+  const [cartData] = useContext(CartContext);
+  const [, appActions] = useContext(AppContext);
+
+  return useMemo(() => {
```

Comment:
I understood why you wrapped this with `useMemo`. I was doing the same previously. But this has no benefit here. Because the dependencies here are prone to change more often which makes this callback called always all time. 

Better remove `useMemo` here